test/nvidia-cuda/install_all_options.sh

```diff
@@ -8,8 +8,14 @@ source dev-container-features-test-lib
 # Check installation of libcudnn8
 check "libcudnn.so.8" test 1 -eq "$(find /usr -name 'libcudnn.so.8' | wc -l)"
 
+# Check installation of libcudnn8-dev
```

Comment:
I don't expect these checks to be pass as the Feature options are `false` by default, you would need to update the following scenario 👇  

https://github.com/devcontainers/features/blob/50a7b160f0c7ea1956e7897f5843ef57332e6141/test/nvidia-cuda/scenarios.json#L2-L10

Also, you'd need to rename the scenario ^ name to `install_all_options`

Reply:
Ref: https://github.com/devcontainers/features/actions/runs/6311978556/job/17158820859?pr=699#step:4:20

Reply:
In https://github.com/devcontainers/features/pull/699/commits/58ae521c84a715d1856bbe05473f621e56015c68, I fixed scenarios.json to pass this test.